### PR TITLE
Prepare some settings for the Developer Preview

### PR DESCRIPTION
**Pull request by LefterisJP**
- Lower settle and reveal timeout
- Switch to solc `0.4.16`
- Add a `contract_version` constant public variable to all contracts. This should be bumped in all contracts as soon as there is a change in one of them.

### CODE REVIEW

**Comment by konradkonrad** [comment]
Can you also adjust the solc version in
`docker/build.Dockerfile:38`
`tools/testnet/files/dockerfiles/raiden/Dockerfile:4`

**Comment by LefterisJP** [comment]
@konradkonrad yep I will. I just got a dejavu :)

File: raiden/settings.py [hunk 1]
```diff
@@ -3,6 +3,7 @@
 
 INITIAL_PORT = 40001
 
+RAIDEN_VERSION = "0.0.6"  # Keep in sync with version in setup.py
```
Comment (konradkonrad) [inline raiden/settings.py]: we don't need this. use `raiden.utils.get_system_spec()['raiden']` or this line directly: https://github.com/raiden-network/raiden/blob/master/raiden/utils/__init__.py#L160

Comment (LefterisJP) [reply raiden/settings.py]: oh thanks. Will do that now.

**Pull request merged**